src/Tribe/Views/V2/Messages.php

```diff
@@ -90,7 +90,14 @@ public function __construct( $render_strategy = null, array $messages = [] ) {
 	 */
 	public static function for_key( $key, ...$values ) {
 		$map = [
-			'no_results_found'                 => __( 'There were no results found.', 'the-events-calendar' ),
+			'no_results_found'                 => __(
+				'There were no results found.',
+				'the-events-calendar'
+			),
+			'month_no_results_found'           => __(
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 35afb13866113225a595c537bda9a249</sub>